Session2.Rmd

```diff
@@ -44,18 +44,23 @@ The `st_geometry_type()` function gives us information about the geometry type,
 st_geometry_type(boundary_Delft)
 ```
 
-The `st_crs()` function gives us the coordinate reference system used by the shapefile, which in this case is `WGS84`. This is not what we want, so we will use the `st_transform()` function to reproject the file to the right CRS. For the `crs` argument we can use the EPSG code of the CRS we want to use. For the Netherlands, we want to use the Amersfort / RD New projection, which has the EPSG code 28992.
+The `st_crs()` function gives us the coordinate reference system used by the shapefile, which in this case is `WGS84`, which has the unique reference code `EPSG: 4326`. The `st_bbox()` function shows the extent of the layer. As `WGS84` is a **geographic CRS**, the extent of the shapefile is displayed in degrees.
```

Comment:
The `st_crs()` function gives us the coordinate reference system (CRS) used by the shapefile [...]

Reply:
Updated. Thanks.